.protolint.yml

```diff
@@ -0,0 +1,174 @@
+---
+# Lint directives.
+lint:
+  # # Linter files to ignore.
+  # ignores:
+  #   - id: MESSAGE_NAMES_UPPER_CAMEL_CASE
+  #     files:
+  #       # NOTE: UNIX paths will be properly accepted by both UNIX and Windows.
+  #       - _example/proto/simple.proto
+  #   - id: ENUM_NAMES_UPPER_CAMEL_CASE
+  #     files:
+  #       - path/to/foo.proto
+
+  # # Linter files to walk.
+  # files:
+  #   # The specific files to exclude.
+  #   exclude:
+  #     # NOTE: UNIX paths will be properly accepted by both UNIX and Windows.
+  #     - path/to/file
+
+  # # Linter directories to walk.
+  # directories:
+  #   # The specific directories to exclude.
+  #   exclude:
+  #     # NOTE: UNIX paths will be properly accepted by both UNIX and Windows.
+  #     - path/to/dir
+
+  # Linter rules.
+  # Run `protolint list` to see all available rules.
+  rules:
+    # Determines whether or not to include the default set of linters.
+    no_default: true
+
+    # Set the default to all linters. This option works the other way around as no_default does.
+    # If you want to enable this option, delete the comment out below and no_default.
+    # all_default: true
+
+    # The specific linters to add.
+    add:
+      - FIELD_NAMES_LOWER_SNAKE_CASE
+      - MESSAGE_NAMES_UPPER_CAMEL_CASE
+      - MAX_LINE_LENGTH
+      - INDENT
+      # - SERVICE_NAMES_END_WITH
+      - FIELD_NAMES_EXCLUDE_PREPOSITIONS
+      - MESSAGE_NAMES_EXCLUDE_PREPOSITIONS
+      - FILE_NAMES_LOWER_SNAKE_CASE
+      - IMPORTS_SORTED
+      - PACKAGE_NAME_LOWER_CASE
+      - ORDER
+      - MESSAGES_HAVE_COMMENT
+      - SERVICES_HAVE_COMMENT
+      - RPCS_HAVE_COMMENT
+      - FIELDS_HAVE_COMMENT
+      - PROTO3_FIELDS_AVOID_REQUIRED
+      - PROTO3_GROUPS_AVOID
+      # - REPEATED_FIELD_NAMES_PLURALIZED
+      - ENUMS_HAVE_COMMENT
+      - ENUM_FIELDS_HAVE_COMMENT
+      - SYNTAX_CONSISTENT
+      - RPC_NAMES_UPPER_CAMEL_CASE
+      # - FILE_HAS_COMMENT
+      - QUOTE_CONSISTENT
+
+    # # The specific linters to remove.
+    # remove:
+    #   - RPC_NAMES_UPPER_CAMEL_CASE
+
+  # Linter rules option.
+  rules_option:
+    # MAX_LINE_LENGTH rule option.
+    max_line_length:
+      # Enforces a maximum line length
+      max_chars: 120
```

Comment:
Optional: Are we sure about 120? 120 is close to uncomfortable length for laptops

Reply:
We're sticking with 120 for now